html/includes/functions.inc.php

```diff
@@ -1696,3 +1696,23 @@ function get_zfs_pools($device_id)
 
     return array();
 }
+
+/**
+ * Returns the sysname of a device with a html line break prepended.
+ * if the device has an empty sysname it will return device's hostname instead
+ * And finally if the device has no hostname it will return an empty string
+ * @param device array
+ * @return string
+ */
+function add_sysname_or_hostname($device)
```

Comment:
Maybe `return_sysname_or_hostname`?

Reply:
How about `get_device_name()`

Reply:
I'm wondering if we pass a $html = false flag to it and it can then be re-used in other places.

Reply:
I removed the html bits altogether.